gramps/gen/lib/serialize.py

```diff
@@ -48,6 +48,20 @@ class DataDict(dict):
     object interface.
     """
 
+    def __init__(self, data=None):
+        """
+        Wrap a data dict (raw data) or object
+        with an attribute API. If data is an
+        object, we use it to get the attributes.
+        """
+        if data is None:
+            super().__init__()
+        elif isinstance(data, dict):
+            super().__init__(data)
+        else:
+            super().__init__()
+            self["_object"] = data
```

Comment:
You are initialising _object without knowing it really is a gramps object. All you know is it’s not None or a dict. Is it worth adding a check or do we trust all clients?

Reply:
I don't think it would be worth it. But it would be worth adding some type checking when we get to that.